### ISSUE TITLE
Error in Supervisor Core log after update 2021.12.0

### ISSUE THREAD

**Issue by BebeMischa** [PiotrMachowski/Home-Assistant-custom-components-Google-Keep#20]
HA supervisor core log is asking me friendly to contact you, so I do listen bravely ;-)

This is, what it has to say:

```
2021-12-11 23:52:50 WARNING (MainThread) [homeassistant.helpers.entity] Entity sensor.google_keep_12836a5845 (<class 'custom_components.google_keep.sensor.GoogleKeepSensor'>) implements device_state_attributes. Please report it to the custom component author.
```

Thank you very much for having a look at it.

**Comment by BebeMischa** [comment]
@PiotrMachowski 

If it helps, those guys had the same issue and got it fixed. Maybe look over there for some idea ;-)
https://github.com/tomaae/homeassistant-openmediavault/issues/63

**Comment by PiotrMachowski** [comment]
It is an easy fix, but I have to find time to do it :D

**Comment by BebeMischa** [comment]
Don't hurry, be happy ;-)

**Comment by BebeMischa** [comment]
Hey Piotr, are you stil alive? ;-)

**Comment by PiotrMachowski** [comment]
Kinda ;) I still remember about this thing, I plan to apply this fix to all of my integrations

**Comment by PiotrMachowski** [comment]
Fixed in v1.1.2
